tests/tests_rest_alerts.py

```
@@ -136,6 +138,44 @@ def test_get_alerts_filter_should_show_newly_created_alert_for_administrator(sel
+        alert_title = f'title{uuid4()}'
+        body = {
+            'alert_title': alert_title,
+            'alert_severity_id': 4,
+            'alert_status_id': 3,
+            'alert_customer_id': 1
+        }
+        response = self._subject.create('/alerts/add', body).json()
+        response = self._subject.get(f'/api/v2/alerts/{_IDENTIFIER_FOR_NONEXISTENT_OBJECT}')
+        self.assertEqual(404, response.status_code)
+
+    def test_create_alert_should_return_403_when_user_has_no_permission_to_alert(self):
```

Ruff (0.8.2)

Redefinition of unused `test_create_alert_should_return_403_when_user_has_no_permission_to_alert` from line 97

(F811)